README.md

```diff
@@ -14,19 +14,18 @@ MixedRealityToolkit-Unity uses code from the base [MixedRealityToolkit](https://
 > Learn more about [Windows Mixed Reality](https://www.microsoft.com/en-gb/windows/windows-mixed-reality) here
 
 [github-rel]:                  https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/latest
-[github-rel-badge]:            https://img.shields.io/github/tag/microsoft/MixedRealityToolkit-unity.svg?style=flat-square&label=latest%20master%20branch%20release&colorB=007ec6
```

Comment:
We should keep this automatic badge updating URL in the README.

Reply:
We _should_ but for this instance of the release we want to point people to the "WMR" flavor for this once

Reply:
True, but the HoloLens one should still point to the "latest", right?

Reply:
Good point

Reply:
Updated the HoloLens section to use the latest.